dmoj/problem.py

```diff
@@ -266,14 +267,15 @@ def _run_generator(self, gen, args=None):
             time_limit = gen.time_limit or time_limit
             memory_limit = gen.memory_limit or memory_limit
             compiler_time_limit = gen.compiler_time_limit or compiler_time_limit
+            cached = gen.get('cached', True)
```

Comment:
When would you want something not to be cached?

Reply:
If you are testing the generator caching it seems wasteful.

Also because there's currently a bug with the caching system, and this is a bit of a hotfix until I can determine the real cause of the bug

Reply:
Update: the "bug" is not a result of the caching system, but the way it was being called.